src/components/Calculator.js

```diff
@@ -1,254 +1,122 @@
-
-          <button
-            type="button"
-            className="option orange"
-            value="X"
-          >
-            <span>
-              X
-
-            </span>
-
-          </button>
-
-          <button
-            type="button"
-            className="option"
-            value="4"
-          >
-            <span>
-              4
-
-            </span>
-
-          </button>
-
-          <button
-            type="button"
-            className="option"
-            value="5"
-          >
-            <span>
-              5
-
-            </span>
-
-          </button>
-
-          <button
-            type="button"
-            className="option"
-            value="6"
-          >
-            <span>
-              6
-
-            </span>
-
-          </button>
-
-          <button
-            type="button"
-            className="option orange"
-            value="-"
-          >
-            <span>
-
-              -
-            </span>
-
-          </button>
-
-          <button
-            type="button"
-            className="option"
-            value="1"
-          >
-            <span>
-              1
-
-            </span>
-
-          </button>
-
-          <button
-            type="button"
-            className="option"
-            value="2"
-          >
-            <span>
-              2
-
-            </span>
-
-          </button>
-
-          <button
-            type="button"
-            className="option"
-            value="3"
-          >
-            <span>
-              3
-
-            </span>
-
-          </button>
-
-          <button
-            type="button"
-            className="option orange"
-            value="+"
-          >
-            <span>
-
-              +
-            </span>
-
-          </button>
-
-          <button
-            className="option zero"
-            type="button"
-            value="0"
-          >
-            <span>
-              0
-
-            </span>
-
-          </button>
-
-          <button
-            className="option"
-            type="button"
-            value="."
-          >
-            <span>
-              .
-
-            </span>
-
-          </button>
-
-          <button
-            className="option orange"
-            type="button"
-            value="="
-          >
-            <span>
-
-              =
-
-            </span>
+    this.state = {};
+  }
 
-          </button>
+    onClickHandler = (event) => {
+      this.setState((prevState) => calculate(prevState, event.target.value));
+    };
+
+    formatOutput = () => {
+      const { total, next, operation } = this.state;
+      const display = (total || '') + (operation || '') + (next || '');
+
+      return display || '0';
+    };
+
+    render() {
+      return (
+        <div className="calculator">
+          <div className="output">
+            <p>
+              {' '}
+              { this.formatOutput() }
+              {' '}
+
+            </p>
```

Comment:
- [ ] Please, handle the division by zero error. When I divided a number by zero, I got this browser error `Error: [big.js] Division by zero`. Please, let's display an error message on the calculator screen instead to tell users that they can't divide by zero.
- [ ] The same thing is applicable to when I mistakenly double-clicked an operator, I received this browser error, `TypeError: one.times is not a function`. Please, let's fix it.
- [ ] The Licence link in your README is broken as well. Please, let's make our README professional at all times.

Reply:
Hey @george-shammar,

Thanks for reviewing my work.
I implemented and fixed all requirements.

Thanks.